static/src/javascripts/projects/journalism/views/campaignForm.html

```diff
@@ -81,9 +81,11 @@ <h4 class="campaign--snippet__headline"> <%= data.title %></h4>
                 <% } else if(field.type === "file") { %>
                 <input name="field_<%=field.id%>"
                        type="file"
-                       accept="image/*, .pdf, .doc, .xml, .docx"
+                       accept="image/*, ,video/*, .pdf"
+                       multiple="multiple"
```

Comment:
Is `multiple` intended to work around the 6MB limit?

Reply:
No it doesn't unfort. They have to add up to under 6MB. which is what I was hoping to get across with the `maximum total` label

Reply:
I'd hold off adding that attribute for now then, if we already know of the limit with one single image.

Reply:
Fair enough will remove multiple 👍